tests/sparseml/transformers/sparsification/modification/conftest.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from copy import deepcopy
+
+import pytest
+from transformers import AutoConfig, AutoModel
+
+from accelerate import init_empty_weights
+from sparseml.transformers import SparseAutoConfig, SparseAutoModelForCausalLM
+from sparseml.transformers.sparsification.modification import modify_model
+
+
+@pytest.fixture
+def mistral_zoo_model():
+    stub = "zoo:mistral-7b-evolcodealpaca_mistral_pretrain-pruned50_quantized"
```

Comment:
Should we find smaller models to use here? Even if the weights aren't downloading, it is a lot of memory to allocate

Reply:
The crux is that this test tests sparsezoo models specifically, and we do not have mistral models smaller than 7b there (or in general afaik)